26/quickstart-docker.html

```diff
@@ -0,0 +1,204 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
You can delete this `quickstart-docker.html` file, we just need `quickstart.html`